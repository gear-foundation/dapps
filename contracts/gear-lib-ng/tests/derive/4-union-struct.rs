use gear_lib::StorageProvider;

#[derive(StorageProvider)]
struct Contract;

fn main() {}
