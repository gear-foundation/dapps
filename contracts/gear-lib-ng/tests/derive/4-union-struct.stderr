error: `StorageProvider` deriving does nothing for unit (empty) structs, add some fields
 --> tests/derive/4-union-struct.rs:4:1
  |
4 | struct Contract;
  | ^^^^^^^^^^^^^^^^
